resources/upgrade-notification.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+FROM_VERSION="${1}"
+TO_VERSION="${2}"
+
+# print major upgrade notification if TO_VERSION is equal or higher than 12.5-1 and FROM_VERSION is lower than 12.5-1
+if [ "12.5-1" == "$(printf "%s\\n12.5-1" "${TO_VERSION}" | sort -n | head -n1)" ] && [ "12.5-1" != "$(printf "%s\\n12.5-1" "${FROM_VERSION}" | sort -n | head -n1)" ]; then
+    printf "You are starting a major upgrade of the PostreSQL dogu (from %s to %s)!\\n" "${FROM_VERSION}" "${TO_VERSION}"
+    echo "Please consider a backup before doing so, e.g. via the Backup dogu!"
+    echo "During the upgrade process a full database dump will be created."
+    echo "Therefore please make sure your harddrive has at least as much free space left as your PostgreSQL database currently occupies!"
```

Comment:
Given that a DB dump takes _more_ space than the actual data (bcs of textual DDLs and so on):
Is there a way to easily provide the user with an actual number or a command that would do so?

Reply:
I got the impression that the dump takes less space than the actual data in my tests. We need to talk about this..

Reply:
Yea but this has rather a low prio. You know the kind of "oh that's useful!"